src/components/header/BeautyDropdown.jsx

```diff
@@ -0,0 +1,330 @@
+import IconButton from '@mui/material/IconButton';
+import Tooltip from '@mui/material/Tooltip';
+import { styled } from '@mui/material/styles';
+import { makeStyles } from '@material-ui/core';
+import Grid from '@mui/material/Grid';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles(() => ({
+  catContainerBeauty: {
+    paddingTop: '1.65em',
+    color: '#282c3f',
+    fontSize: '14px',
+    letterSpacing: '0.5px',
+    fontFamily:
+      'Whitney, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Helvetica, Arial, sans-serif',
+    fontWeight: '500',
+    textTransform: 'uppercase',
+    '&:hover ': {
+      borderBottom: '4px solid #0db7af',
+    },
+  },
+}));
+
+const MenuBold = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  paddingTop: '2px',
+  paddingBottom: '2px',
+  paddingRight: '6px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
+  color: '#3e4152',
+  '&:hover': {
+    color: '#3e4152',
+    fontWeight: '700',
+    background: 'white',
+  },
+}));
+
+const MenuBold2 = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  paddingTop: '2px',
+  paddingBottom: '2px',
+  paddingRight: '6px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
+  color: '#3e4152',
+  '&:hover': {
+    color: '#3e4152',
+    fontWeight: '700',
+    background: 'rgba(245,245,246,.4)',
+  },
+}));
+
+const Heading = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
+  color: '#0db7af',
+  '&:hover': {
+    color: '#0db7af',
+    background: '#fff',
+  },
+}));
+
+const Heading2 = styled(MenuItem)(() => ({
+  fontSize: '14px',
+  fontFamily:
+    ' Whitney,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif',
+  color: '#0db7af',
+  '&:hover': {
+    color: '#0db7af',
+    background: 'rgba(245,245,246,.4)',
+  },
+}));
+
+function ColoredLine({ color }) {
+  return (
+    <hr
+      style={{
+        color,
+        backgroundColor: color,
+        width: '185px',
+      }}
+    />
+  );
+}
+
+export default function BeautyDropdown() {
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', alignItems: 'center', textAlign: 'center' }}>
+        <Tooltip>
+          <IconButton
+            onClick={handleClick}
+            className="nav_items"
+            onMouseOver={handleClick}
+            size="small"
+            sx={{ ml: 2 }}
+            aria-controls={open ? 'account-menu' : undefined}
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
```

Comment:
Instead of checking for undefined you can use a init value for the open in state and use that let's say false in this case!